hack/addons/kustomize/nutanix-storage-csi/kustomization.yaml.tmpl

```diff
@@ -14,7 +14,5 @@ helmCharts:
   repo: https://nutanix.github.io/helm-releases/
   releaseName: nutanix-csi-storage
   version: ${NUTANIX_STORAGE_CSI_CHART_VERSION}
-  valuesFile: helm-values.yaml
   includeCRDs: true
   skipTests: true
-  namespace: nutanix-system
```

Comment:
What is this change for?

Reply:
There are no `helm-values.yaml` here and the same answer as above about the namespace.